test/unit/video.test.js

```diff
@@ -45,6 +46,44 @@ QUnit.test('should return a video player instance', function(assert) {
   player2.dispose();
 });
 
+QUnit.test('should log about already initalized players if options already passed',
```

Comment:
I think we'd ultimately want this test to use `sinon.spy(log, 'warn')` instead and to use the `TestHelpers.makePlayer()` function to create a mock player, but I don't think it should stop the PR from going in since there is time-sensitivity here.